css/excercise_one/part-c/css/excercise_one.css

```diff
@@ -0,0 +1,512 @@
+}
+
+.mt-xxl {
+  margin-top: 300px;
+}
+
+.mt-ng {
+  margin-top: -20px;
+  margin-right: 0;
+  margin-bottom: 0;
+  margin-left: 0;
+}
+
+.mr-sm {
+  margin-right: 8px;
+}
+
+.mr-lg {
+  margin-right: 70px;
+}
+
+.mb-0 {
+  margin-bottom: 0;
+}
+
+.mb-xs {
+  margin-bottom: 8px;
+}
+
+.mb-sm {
+  margin-bottom: 20px;
+}
+
+.mb-md {
+  margin-bottom: 50px;
+}
+
+.mb-lg {
+  margin-bottom: 70px;
+}
+
+.mb-xl {
+  margin-bottom: 80px;
+}
+
+.mb-ng {
+  margin-top: 0px;
+  margin-right: 0;
+  margin-bottom: -65px;
+  margin-left: 0;
+}
+
+.ml-sm {
+  margin-left: 8px;
+}
+
+.ml-md {
+  margin-left: 20px;
+}
+
+.ml-lg {
+  margin-left: 70px;
+}
+
+.my-lg {
+  margin-top: 200px;
+  margin-bottom: 200px;
+}
+
+.text-right {
+  text-align: right;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.text-left {
+  text-align: left;
+}
+
+.text-uppercase {
+  text-transform: uppercase;
+}
+
+.font-weight-light {
+  font-weight: 300;
+}
+
+.font-weight-medium {
+  font-weight: 400;
+}
+
+.font-weight-bold {
+  font-weight: 500;
+}
+
+.font-weight-extra-bold {
+  font-weight: 600;
+}
+
+.text-xs {
+  font-size: 14px;
+}
+
+.text-sm {
+  font-size: 16px;
+}
+
+.text-md {
+  font-size: 20px;
+}
+
+.text-lg {
+  font-size: 30px;
+}
+
+.text-xl {
+  font-size: 35px;
+}
+
+.text-xxl {
+  font-size: 48px;
+}
+
+.text-xxxl {
+  font-size: 60px;
+}
+
+.line-height-sm {
+  line-height: 19.5px;
+}
+
+.line-height-md {
+  line-height: 37.8px;
+}
+
+.line-height-lg {
+  line-height: 52px;
+}
+
+.line-height-xlg {
+  line-height: 65px;
+}
+
+.d-inline-block {
+  display: inline-block;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.flex-column {
+  flex-direction: column;
+}
+
+.justify-content-start {
+  justify-content: flex-start;
+}
+
+.justify-content-center {
+  justify-content: center;
+}
+
+.justify-content-end {
+  justify-content: flex-end;
+}
+
+.justify-content-between {
+  justify-content: space-between;
+}
+
+.justify-content-around {
+  justify-content: space-around;
+}
+
+.align-items-center {
+  align-items: center;
+}
+
+.position-relative {
+  position: relative;
+}
+
+.position-absolute {
+  position: absolute;
+}
+
+.top-0 {
+  top: 0;
+}
+
+.right-0 {
+  right: 0;
+}
+
+.bottom-0 {
+  bottom: 0;
+}
+
+.bottom-20px {
+  bottom: 20px;
+}
+
+.left-0 {
+  left: 0;
+}
+
+.left-365px {
+  left: 365px;
+}
+
+.rounded {
+  border-radius: 5px;
+}
+
+.top-rounded {
+  border-top-left-radius: 20px;
+  border-top-right-radius: 20px;
+}
+
+.letter-spacing-negative {
+  letter-spacing: -1px;
+}
+
+.gap-1 {
+  gap: 10px;
+}
+
+.gap-2 {
+  gap: 20px;
+}
+
+.gap-3 {
+  gap: 40px;
+}
+
+.gap-4 {
+  gap: 70px;
+}
+
+.gap-5 {
+  gap: 200px;
+}
+
+.dark-blue-gradient {
+  background: linear-gradient(46.99deg, #0F58B7 -4.4%, #62D9FF 120.42%);
+}
+
+.light-blue-gradient {
+  background: linear-gradient(46.77deg, #0F99B7 -0.79%, #69ECDC 131.52%);
+}
+
+.background-clip-text {
+  -webkit-background-clip: text;
+}
+
+.text-transparent {
+  color: transparent;
+}
+
+.div-gradient {
```

Comment:
what is div-gradient? if this is custom class as I believe it is, lets put more descriptive name to let the person know what this is for